### PR TITLE
Tests/create tests

### LINKED ISSUE #89: Create Tests

### PR DESCRIPTION
**Pull request by Njord201**

### CODE REVIEW

**Review by mathieurobert1 — CHANGES_REQUESTED** [review]
coding style error and tmp files not deleted with fclean

File: Makefile [hunk 1]
```diff
@@ -5,6 +5,96 @@
 ## Makefile of the project
 ##
 
+SRC = 		server/src/parsing.c 				\
+			server/src/lists/channels_list.c	\
+			server/src/lists/clients_list.c 	\
+			server/src/lists/teams_list.c 		\
```
Comment (theophile-jr) [inline Makefile:11]: I'd rather avoid changing that much the makefile, we are close to the delivery.

File: cli/Makefile [hunk 2]
```diff
@@ -26,15 +26,7 @@ CFLAGS = -Wall -Wextra -Werror
 INC = -I./include -I../shared/include \
 	-I../libs/myteams/ -L ../libs/myteams/ -lmyteams
 
-TEST_FILES =
-
```
Comment (theophile-jr) [inline cli/Makefile:30]: I'd rather avoid changing that much the makefile, we are close to the delivery.

File: server/Makefile
Comment (theophile-jr) [inline server/Makefile:1]: I'd rather avoid changing that much the makefile, we are close to the delivery.

**Review by theophile-jr — CHANGES_REQUESTED** [review]

**Comment by theophile-jr** [comment]
+ coding style

**Pull request merged**